android/app/src/main/AndroidManifest.xml

```diff
@@ -67,6 +67,21 @@
             </intent-filter>
         </activity>
 
+        <receiver android:name=".LargeWidgetProvider" android:label="@string/large_widget" android:exported="false">
```

Comment:
We should have only one forecast widget that scales to different sizes ->  User should have only one widget to choose.

Reply:
Where is that specified?

I do not think that is a good idea. The differences are quite big between Type A and Type B widgets.

It is not obvious to the user that the widgets can look so differently, when they can choose only from one widget at start.

Reply:
@AnttiRumpunen, can you comment on this?

Reply:
It has been discussed in meetings, maybe not written in specs.

I think that it works better if user has just one forecast widget to choose, but that's my opinion. It works well in iOS widgets that are similar and in this Google demo

https://developer.android.com/static/images/appwidgets/size-range.gif

from 

https://developer.android.com/develop/ui/views/appwidgets/layouts

Reply:
Surely it can be done in Android.

The question is, if it is a good idea from UX point of view.

ping @AnttiRumpunen @helgavisuri

Reply:
The users cannot know that they can have a large widget, if they see only the small widget in the preview.

Reply:
That's true at least with most Android launchers. On the other hand it is more clear if user have only three choices  vs. 8 choices (3 x forecast, 3 x one day warnings, 2 x 5 day warnings). In the future there might be even more widgets (at least observations).

Reply:
OK, it was decided to keep current implementation. But please rename providers so that LargeWidgetProvider -> LargeForecastProvider and so on. Then we have clear naming when we implement different kind of widgets.

Reply:
OK.
But let's do the renaming with the next review (Type-C widget).
Otherwise I will run into trouble with the Type-C branch now.

Reply:
That's fine